docs/markdown/Dependencies.md

```diff
@@ -317,6 +317,16 @@ dep = dependency('appleframeworks', modules : 'foundation')
 
 These dependencies can never be found for non-OSX hosts.
 
+## atomic (stdatomic)
+
+*(added 1.5.0)*
+
+Provides access to the atomic operations library. This first attempts
+to look for a valid atomic external library before trying to fallback
+to what is provided by the libc.
```

Comment:
```suggestion
to what is provided by the C runtime libraries.
```

but let's fix that on merge to avoid delaying this more.

Reply:
Well, since we forgot to re-review this a while back we do need to bump the new-since version number yet again... :)

Other than that though I do think this looks quite reasonable.